subscription-service/src/test/kotlin/com/egm/stellio/subscription/service/NotificationServiceTests.kt

```diff
@@ -404,4 +422,120 @@ class NotificationServiceTests {
 
         verify(1, postRequestedFor(urlPathEqualTo("/notification")))
     }
+
+    @Test
+    fun `it should notify the subscriber and return entities without system attributes if sysAttrs is false`() = runTest {
+
+        val payload = mapOf(
```

Comment:
btw, when don't you use the `gimmeRawSubscription` helper function as it is done in other tests?

Reply:
I thought a subscription must be created with an entity in this case and that method creates ones without entities, that's why I didn't use it

Reply:
in this case it can be used , since sysAttrs is by default false , but when it needs to be set to true I can't use it.

Reply:
you can use it and then create a copy a subscription, overriding what you need to overwrite (it is already done in many other tests in the class)